tests/lax_numpy_ufuncs_test.py

```diff
@@ -87,6 +87,7 @@ def _jnp_ufunc_props(name):
 
 broadcast_compatible_shapes = [(), (1,), (3,), (1, 3), (4, 1), (4, 3)]
 nonscalar_shapes = [(3,), (4,), (4, 3)]
+with_zerodim_shapes = nonscalar_shapes + [(0, 3), (4, 0)]
```

Comment:
minor nit: `zerodim` implies to me that `arr.ndim == 0`. Can we do similar to `lax_numpy_test.py` and call these `empty_shapes = [(0, 3), (4, 0)]` and then define `all_shapes = nonscalar_shapes + empty_shapes`?